core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -3,13 +3,13 @@
 package chisel3.internal.firrtl
 
 import firrtl.{ir => fir}
-
 import chisel3._
 import chisel3.internal._
 import chisel3.internal.sourceinfo.SourceInfo
 import chisel3.experimental._
 import _root_.firrtl.{ir => firrtlir}
 import _root_.firrtl.PrimOps
+import chisel3.DontCare._parent
```

Comment:
```suggestion
```
This import appears to be unused and is a weird one that probably should never be used 🧐

Reply:
oops, blaming this one on intelliJ >_>